### ISSUE TITLE
The "Focus Chat" keybinding does not automatically focus the chat message input element if the Chat sidebar tab is already active

### ISSUE THREAD

**Issue by Norc** [foundryvtt/foundryvtt#13522]
### What happened?

As of Version 13, the Focus Chat kebyinding (`SHIFT + C` by default) no longer automatically inserts the cursor into the chat message text box if the Chat sidebar tab is already active. I confirmed that it did insert the cursor as expected in 12.343. 

Unless I'm missing something, this bug means that there is currently no way to reliably enter a chat message using only the keyboard without the use of a macro, so this is a bit of an accessibility issue.

### What ways of accessing Foundry can you encounter this issue in?

- [x] Native App (Electron)
- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

1. In a clean world, ensure that the Chat tab is active
2. Press `SHIFT + C `

**Expected**: The cursor moves to the Chat Message input box to allow input
**Current**: The cursor does not move to the Chat Message input box to allow input

### What core version are you reporting this for?

13.350

### Relevant log output

```shell

```

### Bug Checklist

- [x] The issue occurs while all Modules are disabled
